observation_portal/sciapplications/migrations/0005_auto_20210628_0304.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.23 on 2021-06-28 03:04
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sciapplications', '0004_remove_timerequest_instrument'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='timerequest',
+            options={'ordering': ('semester',)},
```

Comment:
oh I vaguely remember having to mess with the ordering so that's probably why this is here